config/config_windows_test.go

```diff
@@ -11,7 +11,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package config
+package config_test
```

Comment:
Why do you need this?

Reply:
It prevents a import cycle. The config tests import various discovery packages and those discover packages register with config.